backend/src/schema/types/type/Report.gql

```diff
@@ -3,13 +3,19 @@ type Report {
   createdAt: String!
   updatedAt: String!
   rule: ReportRule!
-  disable: Boolean!
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-02-13T08:40:04Z" title="Thursday, February 13th 2020, 9:40:04 am +01:00">Feb 13, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -3,13 +3,19 @@ type Report {
   createdAt: String!
   updatedAt: String!
   rule: ReportRule!
-  disable: Boolean!
```


Like above ☝🏼, I strongly disagree with that !!!
Please store decisions in the database! 😎